common-lib/src/main/java/com/google/cloud/tools/intellij/BasePluginInfoService.java

```diff
@@ -41,6 +41,7 @@
   private final String userAgent;
   private final IdeaPluginDescriptor plugin;
   private final FlagReader flagReader;
+  private static final String PLUGIN_NAME_EXTERNAL = "gcloud-intellij";
```

Comment:
this change belongs in a separate PR.

@chanseokoh I'm actually concerned that the new scheme loses the IDE platform the plugin is being used from.  When we interact with APIs directly we specify which IDE the plugin is in (Android Studio vs IntelliJ).  Can we just add this information to the user-agent?